tests/smoke/cluster_test.go

```diff
@@ -61,7 +61,7 @@ func testCluster(t *testing.T) {
 	t.Run("AllNodesRunning", testAllNodesRunning)
 	t.Run("AllResourcesCreated", testAllResourcesCreated)
 	t.Run("AllPodsRunning", testAllPodsRunning)
-	t.Run("KillAPIServer", testKillAPIServer)
```

Comment:
Either add a todo where you commented this saying you should re-enable the test, or delete the line entirely. It is in git history anyways so there is no reason to keep it commented out.

Reply:
done